packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -162,16 +165,28 @@ class _TreeTableState<T extends TreeNode<T>> extends State<TreeTable<T>>
     });
   }
 
+  void _onItemsAnimated() {
+    setState(() {
+      animatingChildren = [];
+      animatingChildrenSet = {};
+    });
+  }
+
   void _onItemPressed(T node) {
     /// Rebuilds the table whenever the tree structure has been updated
     if (!node.isExpandable) return;
     setState(() {
+      animatingNode = node;
       if (node.isExpanded) {
+        animatingChildren = _buildFlatList(node).skip(1).take(30).toList();
```

Comment:
30 seems like a magic number. Please add a comment and store 30 in a const.

Reply:
Done.